src/resources/elements/EtherscanLink/EtherscanLink.scss

```diff
@@ -1,5 +1,6 @@
 .etherscanlink {
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
```

Comment:
better make sure this doesn't bust stuff using this component elsewhere

Reply:
for the current use cases in the app- all looks good.